lib/screens/app/ecosystem/overview.dart

```diff
@@ -201,14 +207,23 @@ class _OverviewState extends State<Overview> {
                     onGet,
                   ),
                 ),
+                buildCategory(
+                    'Guardians',
+                    'Protect your account sharing trust with friends',
+                    'assets/images/harvest.svg',
+                    '',
+                    '',
+                    onGuardians),
```

Comment:
Guardians is already in Settings, should not be here

Reply:
It's OK if this PR simply only provides the service calls - @gery can do the rest, the whole UX is done